common/hogli/manifest.yaml

```diff
@@ -268,8 +262,8 @@ docker:
             - redis
             - kafka
     docker:services:down:
-        cmd: docker compose -f docker-compose.dev.yml down
-        description: Stop Docker infrastructure services
+        cmd: docker compose -f docker-compose.dev.yml down -v
+        description: Stop Docker infrastructure services and remove volumes
```

Comment:
Maybe these should be two separate things? We don't wanna break those places where we weren't expecting to drop volumes

And then we use the new one on the reset script?